configuration/pih/liquibase/liquibase.xml

```diff
@@ -972,4 +972,25 @@
         </sql>
     </changeSet>
 
+    <changeSet id="20240920-migrate-outpatient-peds-form-id" author="ball">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                select count(*) from address_hierarchy_entry where parent_id is NULL and name like 'Haiti';
```

Comment:
We also need a precondition to check  that select form_id from form where uuid = ''58f731af-8f8e-4863-b721-70bca76699c4' is not null.